packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -565,8 +570,7 @@ export class GarbageCollector implements IGarbageCollector {
             // flag in GC options to false.
             this.gcEnabled = this.gcOptions.gcAllowed !== false;
             // The sweep phase has to be explicitly enabled by setting the sweepAllowed flag in GC options to true.
-            // ...unless we're using the TestOverride
-            this.sweepEnabled = this.gcOptions.sweepAllowed === true || testOverrideSweepTimeoutMs !== undefined;
```

Comment:
@markfields I removed the `testOverrideSweepTimeoutMs !== undefined` condition because in the end-to-end tombstone tests, this was setting sweepEnabled to true. Basically, for tombstoning, we need to override sweep timeout and that enabled sweep - this seems wrong.
Will this break anything?

Reply:
So, I think I added it that way to make it trivial to do manual testing. i.e. you could just set the testOverride and not worry about hacking in the gcOptions.  But I don't think that really matters... so this is fine.